src/README.md

```diff
@@ -0,0 +1,6 @@
+Angular Flex-Layout
```

Comment:
I am confused. This is the readme for the source repository... same for months.
We have a separate README that is deployed with the npm release.